src/lib/tools/FileBrowser.svelte

```
@@ -61,14 +59,15 @@
 				keyboard: {
-					f2: () => {}
+					f2: () => {
+					}
 				}
 			},
 			search: {
 				show_only_matches: true,
 				close_opened_onclear: false
 			},
-			sort: function (a: any, b: any) {
+			sort: function(a: any, b: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)